content/applications/inventory_and_mrp/inventory/warehouses_storage/replenishment/reordering_rules.rst

```diff
@@ -59,30 +56,39 @@ If a :abbr:`BoM (Bill of Materials)` does not already exist for the product, sel
 :guilabel:`New` to configure a new :abbr:`BoM (Bill of Materials)`.
 
 .. image:: reordering_rules/bom-smart-button.png
-   :align: center
    :alt: The Bill of Materials smart button on a product form.
 
 Create new reordering rules
 ===========================
 
-To create a new reordering rule, navigate to :menuselection:`Inventory app --> Configuration -->
-Reordering Rules`, then click :guilabel:`New`, and fill out the new line as follows:
+To create a new reordering rule, navigate to
+:menuselection:`Inventory app --> Operations --> Replenishment`, then click :guilabel:`New`,
+and fill out the following fields for the new reordering rule line item:
```

Comment:
Can break **menuselection** midway through the tag, see the **Tip** and **Example** here in the [rst guidelines](https://www.odoo.com/documentation/master/contributing/documentation/rst_guidelines.html#th-character-limit).

```suggestion
To create a new reordering rule, navigate to :menuselection:`Inventory app --> Operations -->
Replenishment`, then click :guilabel:`New`, and fill out the following fields for the new
reordering rule line item:
```